src/Interfaces/code/Code.ts

```diff
@@ -1,11 +1,22 @@
+import SubscriptionDiscount from '../subscription/SubscriptionDiscount';
+import Interest from '../interest/Interest';
+import Gift from '../gift/Gift';
 import Type from './Type';
 
 export default interface Code {
     name?: string;
```

Comment:
It's possible here that we can start removing `?` as this interface may be imported maybe twice when setting up waccCode class etc, so we can use partials, what do you think?